src/loaders/collection-loader.coffee

```diff
@@ -67,7 +70,7 @@ class CollectionLoader extends AbstractLoader
     # Loop over all returned data types and update our local storage to represent any new data.
 
     results = resp['results']
-    keys = _.reject(_.keys(resp), (key) -> key == 'count' || key == 'results')
+    keys = _.without(_.keys(resp), knownResponseKeys...)
```

Comment:
💯

Reply:
Thanks. Also just realized that we can perhaps use `apply` if we ever need to avoid the CoffeeScript 2 syntax (I _think_ the `knownResponseKeys...` is v2)

Reply:
Looks like the spread operator is supported in CoffeeScript 1.

Reply:
Nice! For some reason I thought it was coffeescript 2